paper_io_sim.tex

```diff
@@ -303,8 +303,8 @@
     and to read or write cached data. The Memory Manager
     implements these operations, simulates periodical flushing
     and eviction, and reads or writes to disk when necessary.
-    In case the writethrough strategy is used, the I/O Controller directly
-    writes to disk \tristan{Then how is the written data added to cache?}.
+    In case the writethrough strategy is used, the I/O Controller directly writes to disk, 
+    cache is flushed if needed and written data is added to page cache.
```

Comment:
all the cache is flushed? if not, how? would flushing delay writing this data to cache?

Reply:
this should be "evicted", not "flushed" because we don't have dirty data

Reply:
ok, so all the cached data is evicted with every write?

Reply:
I couldn't find it in documents, but I think we don't necessarily evict all cached data, we only need to evict a required amount